samples/charger_and_events/Kconfig

```
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+mainmenu "Charger and events sample"
```

```suggestion
mainmenu "CHARGER and EVENTS sample"
```
@masz-nordic @wilu-nordic When talking about a sample, should we capitalize CHARGER and EVENTS? If so, please correct all occurrences.

---

Both versions are ok to me.